msteams-platform/resources/removing-tab-margins.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Removing tab margins in Microsoft Teams
+author: laujan
+description: 
+keywords: tab removing margins padding
+ms.topic: reference
+ms.author: lomeybur
+---
+
+# Removing all tab margins (2021 update)
```

Comment:
I'm reluctant to title this document "Remove all tab margins" since - even though that's what _we're_ doing - the call-to-action for developers is to provide their own margins. So the title may be misleading or misunderstood.

Other options:
* Tab margin changes
* Removal of all tab margins
* Teams-provided tab margin changes

Thoughts? Personally, I prefer the first one.

Reply:
Thanks @lokimeyburg. We are going with what you have provided in the document.